lib/stripe/resources/charge.rb

```diff
@@ -10,75 +10,9 @@ class Charge < APIResource
 
     custom_method :capture, http_verb: :post
 
-    def refund(params = {}, opts = {})
-      # Old versions of charge objects included a `refunds` field that was just
-      # a vanilla array instead of a Stripe list object.
-      #
-      # Where possible, we'd still like to use the new refund endpoint (thus
-      # `self.refunds.create`), but detect the old API version by looking for
-      # an `Array` and fall back to the old refund URL if necessary so as to
-      # maintain internal compatibility.
-      if refunds.is_a?(Array)
-        resp, opts = request(:post, refund_url, params, opts)
-        initialize_from(resp.data, opts)
-      else
-        refunds.create(params, opts)
-
-        # now that a refund has been created, we expect the state of this object
-        # to change as well (i.e. `refunded` will now be `true`) so refresh it
-        # from the server
-        refresh
-      end
-    end
-
     def capture(params = {}, opts = {})
-      resp, opts = request(:post, capture_url, params, opts)
-      initialize_from(resp.data, opts)
-    end
-
-    def update_dispute(params = {}, opts = {})
-      resp, opts = request(:post, dispute_url, params, opts)
-      initialize_from({ dispute: resp.data }, opts, true)
-      dispute
-    end
-
-    def close_dispute(params = {}, opts = {})
-      resp, opts = request(:post, close_dispute_url, params, opts)
-      initialize_from(resp.data, opts)
-    end
-
-    def mark_as_fraudulent
-      params = {
```

Comment:
While we considered those deprecated for a long time, we never said it ~anywhere and it's actively documented to this day.
I opened a PR to fix the docs and still think we should remove, but wanted to flag.

Reply:
👍 Thanks!